File: app/src/main/java/com/jetsynthesys/rightlife/ui/drawermenu/FAQData.java
package com.jetsynthesys.rightlife.ui.drawermenu;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class FAQData {

    @SerializedName("faqs")
    @Expose
    private List<Faq> faqs;

    public List<Faq> getFaqs() {
        return faqs;
    }

    public void setFaqs(List<Faq> faqs) {
        this.faqs = faqs;
    }
}

File: app/src/main/java/com/jetsynthesys/rightlife/ui/settings/pojo/FAQData.kt
package com.jetsynthesys.rightlife.ui.settings.pojo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class FAQData {
    @SerializedName("_id")
    @Expose
    var id: String? = null

    @SerializedName("title")
    @Expose
    var title: String? = null

    @SerializedName("details")
    @Expose
    var details: ArrayList<FAQDetails>? = ArrayList()

    @SerializedName("order")
    @Expose
    var order: Int? = null

    @SerializedName("createdAt")
    @Expose
    var createdAt: String? = null

    @SerializedName("updatedAt")
    @Expose
    var updatedAt: String? = null

    @SerializedName("__v")
    @Expose
    var v: Int? = null
}